packages/react-core/src/components/ChipGroup/examples/ChipGroup.md

```diff
@@ -69,174 +72,81 @@ class SingleChip extends React.Component {
         <Chip key="chip4" onClick={() => this.deleteItem('readonlychip')} isReadOnly>
```

Comment:
is `isReadOnly` still necessary? We don't have a variation for that in core anymore. You create a read only chip by omitting the close button, and looks like the react chip component doesn't add a close button if you don't specify `onClick()`, which seems to create all you need for a read only chip.

Reply:
The component currently uses the the  isReadOnly flag to omit the close button.   with the current implementation, if the consumer does not provide a `onClick` function and `isReadOnly === false`, the close button will still be rendered.

Reply:
Ah yep you're right, I'm not sure why I thought it was removed if you didn't pass `onClick()`. Since `onClick()` is only used to define the function to run when the close button is clicked, it seems intuitive that omitting that would omit the close button, but maybe there are other reasons not to do that.